src/http/server.ts

```diff
@@ -782,28 +816,32 @@ export class Server {
    * @return boolean
    *     Returns true if the request targets a static path.
    */
-  protected requestTargetsStaticPath(request: any): boolean {
+  protected requestTargetsStaticPath(serverRequest: ServerRequest): boolean {
```

Comment:
Has the doc block been updated to reflect this?

Reply:
3565ceb